shell/utils/__tests__/formatter.test.ts

```diff
@@ -0,0 +1,88 @@
+import { formatEncryptionSecretNames } from '@shell/utils/formatter';
+
+describe('formatter', () => {
+  const secrets = [
+    {
+      id:       'test5',
+      _type:    'Opaque',
+      data:     { hash: 'test5', 'encryption-provider-config.yaml': 'MTIzNFFhYWEh' },
+      metadata: {
+        name:      'test5',
+        namespace: 'test',
+        state:     {
+          error: false, message: 'Resource is always ready', name: 'active', transitioning: false
+        }
+      }
+    },
+    {
+      id:       'test2',
+      _type:    'Opaque',
+      data:     { hash: 'test', 'encryption-provider-config.yaml': 'MTIzNFFhYWEh' },
+      metadata: {
+        name:      'test2',
+        namespace: 'test',
+        state:     {
+          error: false, message: 'Resource is always ready', name: 'active', transitioning: false
+        }
+      }
+    },
+    {
+      id:       'test4',
+      _type:    'Opaque',
+      data:     { hash: 'test4' },
+      metadata: {
+        name:      'test4',
+        namespace: 'test',
+        state:     {
+          error: false, message: 'Resource is always ready', name: 'active', transitioning: false
+        }
+      }
+    },
+    {
+      id:       'test1',
+      _type:    'Custom',
+      data:     { hash: 'test1', 'encryption-provider-config.yaml': 'MTIzNFFhYWEh' },
+      metadata: {
+        name:      'test1',
+        namespace: 'test',
+        state:     {
+          error: false, message: 'Resource is always ready', name: 'active', transitioning: false
+        },
+      }
+    },
+    {
+      id:       'test3',
+      _type:    'Opaque',
+      data:     { hash: 'test', 'encryption-provider-config.yaml': 'MTIzNFFhYWEh' },
+      metadata: {
+        namespace: 'test',
+        state:     {
+          error: false, message: 'Resource is always ready', name: 'active', transitioning: false
+        }
+      }
+    },
+    {
+      id:       'test6',
```

Comment:
Sorry what's the purpose of the new error test?

Reply:
It is one of the conditions based on which we are filtering out secrets. so it's good to check that case as well